js/lib/bandstructure.js

```diff
@@ -37,7 +37,7 @@
 // Utility 'zip' function analogous to python's, from
 // https://stackoverflow.com/questions/4856717
 
-const Chart = require('chart.js');
+require('chart.js/dist/chart');
```

Comment:
Could you explain why the variable is not used by still need to initialize? I search the file and file line 513 has `Chart`.

Reply:
For version 3.x, the import method has been changed. Yes, we are still need to use Chart function. However, we do not need to init a const.

Reply:
I see, so once import module by `require('chart.js')`, `Chart` function in line `513` is able to be used, correct?

Reply:
require('chart.js/dist/chart') will work.